assignments/prototype-refactor.js

```diff
@@ -7,3 +7,115 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+
+class GameObject {
+    constructor (GameObjectAttributes) {
+    this.createdAt = GameObjectAttributes.createdAt;
```

Comment:
Somewhat of an opinion, but I believe indenting code within blocks promotes readability. You are doing that with `lambda-classes`, so no biggie.